### PR TITLE
Script updates for OpenXR-1.0.22 on Linux aarch64 (ARM64)

### PR DESCRIPTION
**Pull request by spham-amzn**
**Build Command**
```
python3 ../3p-package-scripts/o3de_package_scripts/build_package.py --search_path . OpenXR-1.0.22-rev2-linux-aarch64
```

**Build output**
```
Creating/Updating package: OpenXR-1.0.22-rev2-linux-aarch64
    Adding files to: "./packages/temp/temp_package_OpenXR-1.0.22-rev2-linux-aarch64"
    Created: ./packages/OpenXR-1.0.22-rev2-linux-aarch64.tar.xz
    Created: ./packages/OpenXR-1.0.22-rev2-linux-aarch64.tar.xz.SHA256SUMS
    Created: ./packages/OpenXR-1.0.22-rev2-linux-aarch64.tar.xz.content.SHA256SUMS
    Created: ./packages/OpenXR-1.0.22-rev2-linux-aarch64.PackageInfo.json
    - Validating package: OpenXR-1.0.22-rev2-linux-aarch64 in folder ./packages....
    Package Hash: 858ed7fda2b3f7347aad3fb24b5d768e7ea0f06b2ec0ee17964a0727cf460290 *OpenXR-1.0.22-rev2-linux-aarch64.tar.xz
```

**Upload output**
```
Using bucket: ly-dev-3p-packages
Package: OpenXR-1.0.22-rev2-linux-aarch64 ...
    - Validating package: OpenXR-1.0.22-rev2-linux-aarch64 in folder ./packages....
    - Uploading OpenXR-1.0.22-rev2-linux-aarch64.tar.xz...
    - Uploading OpenXR-1.0.22-rev2-linux-aarch64.tar.xz.SHA256SUMS...
    - Uploading OpenXR-1.0.22-rev2-linux-aarch64.tar.xz.content.SHA256SUMS...
    - Uploading OpenXR-1.0.22-rev2-linux-aarch64.PackageInfo.json...
    - Uploaded package OpenXR-1.0.22-rev2-linux-aarch64.
```

### CODE REVIEW

File: package-system/OpenXR/build_config.json [hunk 1]
```diff
@@ -60,7 +60,8 @@
             ],
             "custom_test_cmd" : [
            ]
-         }
+         },
+	 "Linux-aarch64": "@Linux"
```
Comment (moraaar) [inline package-system/OpenXR/build_config.json:64]: will this just use the x64 binaries for the aarch64? Will that work?

Comment (spham-amzn) [reply package-system/OpenXR/build_config.json:64]: Its an alias for situations where the build settings for the platform are identical. For this library, the aarch64 detection, etc is part of the native project's build scripts.

Comment (spham-amzn) [reply package-system/OpenXR/build_config.json:64]: Here is an example if there is a difference ( https://github.com/o3de/3p-package-source/pull/135  ) 
This case the dependent binary is an aarch64 one specifically.

Comment (moraaar) [reply package-system/OpenXR/build_config.json:64]: >For this library, the aarch64 detection, etc is part of the native project's build scripts.

Nice! Didn't know the library detects it an uses the right build parameters automatically. Thanks. I'll approve.

**Review by moraaar — APPROVED** [review]

**Review by lumberyard-employee-dm — APPROVED** [review]

**Review by cgalvan — APPROVED** [review]

**Pull request merged**